plugin/cr/src/test/java/org/opencds/cqf/ruler/cr/r4/RepositoryServiceTest.java

```diff
@@ -732,6 +733,22 @@ void release_test_condition_missing() {
 		}
 		assertNotNull(noConditionExtension);
 		assertTrue(noConditionExtension.getMessage().contains("Missing condition"));
+		
+		//reset
+		noConditionExtension = null;
+		try {
+			getClient().operation()
+			.onInstance(specificationLibReference)
+			.named("$package")
+			.withNoParameters(Parameters.class)
+			.returnResourceType(Bundle.class)
+			.execute();
+		} catch (UnprocessableEntityException e) {
+			// TODO: handle exception
+			noConditionExtension = e;
+		}
+		assertNotNull(noConditionExtension);
+		assertTrue(noConditionExtension.getMessage().contains("Missing condition"));
```

Comment:
had to move this test here since ingesting the test data caused SQL errors during testing